packages/html-reporter/src/testResultView.tsx

```diff
@@ -57,8 +58,8 @@ export const TestResultView: React.FC<{
       {result.steps.map((step, i) => <StepTreeItem key={`step-${i}`} step={step} depth={0}></StepTreeItem>)}
     </AutoChip>}
 
-    {expected && actual && <AutoChip header='Image mismatch'>
-      <ImageDiff actual={actual} expected={expected} diff={diff}></ImageDiff>
+    {expected && actual && <AutoChip header={`${hasImages ? 'Image' : 'Snapshot'} mismatch`}>
```

Comment:
We leave this block since it is convenient to be able to download the expected/actual/diff attachments regardless of their content type.